### DISCUSSION TITLE
I am trying to add a scroll to top function and my scroll function is not working

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1043] [Q&A]
I use this scroll-to-top function on several sites but for some reason this script will not work: ( i am using a custom code block for this)

<div id="BackToTop">
  <a id="back-to-top" href="#" class="back-to-top" role="button" title="Click to return on the top page" data-toggle="tooltip" 
      data-placement="left">^
  </a>
</div>
```
<script>
  function backToTop() {
    /* Check show or hide the button [back-to-top] */
    $(window).scroll(function () {
      if ($(this).scrollTop() > 50) {
        $('#back-to-top').fadeIn();
      }
      else {
        $('#back-to-top').fadeOut();
      }
    }
                    );
    // scroll body to 0px on click
    $('#back-to-top').click(function () {
      $('body,html').animate({
        scrollTop: 0
      } , 800);
      return false;
    }
                           );
  }
</script>  
```
If I change the CSS for #back-to-top to display:block then I can see the icon and click and the page will scroll to top. So the second function is working as it should. Am I referencing the DOM incorrectly?  And I cannot break the javascript using the web developer so I can trace the execution.

**Comment by meetmandeep** [comment]
I would try adding the script in page header (Page Settings > SEO) rather than a Custom Block to see if that makes a difference.
